secboot/secboot.go

```diff
@@ -131,9 +131,12 @@ type SealKeysParams struct {
 type SealKeysWithFDESetupHookParams struct {
 	// Initial model to bind sealed keys to.
 	Model ModelForSealing
-	// The path to the aux key file (if empty the key will not be
-	// saved)
-	AuxKeyFile string
+}
+
+type KeyFileOrSlotName struct {
```

Comment:
should we swap the names areound SlotNameOrKeyFile?  this probably merits a doc comment? do we want a validate method that errors if SlotName and KeyFile are both set? the other approach is to use an interface and an exported struct and have function that only create a variant with KeyFile set or SlotName set?

Reply:
They must be both set. This type is used for resealing. We have to discovered where the key data can be saved and it could be any of those places. So they have to be both set.

Reply:
Would KeyFileAndSlotName be better? The point is that only one will be used in the end, but we will try the slot name first.